core/unit_test/TestMDRangePolicyConstructors.hpp

```diff
@@ -138,4 +138,50 @@ TEST(TEST_CATEGORY_DEATH, policy_invalid_bounds) {
 }
 #endif
 
+TEST(TEST_CATEGORY, policy_get_tile_size) {
+  constexpr int rank = 3;
+  using Policy    = Kokkos::MDRangePolicy<TEST_EXECSPACE, Kokkos::Rank<rank>>;
+  using tile_type = typename Policy::tile_type;
+
+  std::size_t last_rank =
+      (Policy::inner_direction == Kokkos::Iterate::Right) ? rank - 1 : 0;
+
+  auto default_size_properties =
+      Kokkos::Impl::get_tile_size_properties(TEST_EXECSPACE());
+
+  {
+    int dim_length = 100;
+    Policy policy({0, 0, 0}, {dim_length, dim_length, dim_length},
+                  tile_type{{2, 4, 16}});
```

Comment:
How do any of these parameters affect the test?

Reply:
The third parameter of the MDRangePolicy constructor used here, tile_type{{2, 4, 16}}, allows users to specify the desired tile size for the policy, but that tile size may not be equivalent to the recommended tile size.
The test simply checks that the correct default values are being returned when `tile_size_recommended` or `max_total_tile_size` is called, regardless of the user specified tile size.